### PR TITLE
[RU] Breach Scanner, SCP-939 abilities and new LoadingHints

### PR DESCRIPTION
**Pull request by bladuk**
# **SCP:SL Translation PR Template**

## **Language Name**
Russian

## **State Type of change (New language, changed existing Etc.)**
Changed existing
Added Breach Scanner translation
Added new SCP-939 abilities translation

# **Checklist:**
- [ ] I have tested the translation changes ingame
- [x] I have made sure to not commit changes to multiple translations at the same time, or non-translation repo files
- [x] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  *( Exception to this is when edits has __only__ been made in-browser. If this is the case, also* ***check this box*** *)*
- [x] I have made sure that I'm **not** submitting a duplicate translation 
- [x] I have made sure my addition follows guidelines described in the [CONTRIBUTING.md](https://github.com/northwood-studios/SCPSL-Translations/blob/master/.github/CONTRIBUTING.md) file
- [x] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of **new** translations (Can be tested [Here](https://rextester.com/WDXPS97501)). Also check this box if the change is **not** a new translation
> For more information regarding game translations, please visit our [README](https://github.com/northwood-studios/SCPSL-Translations/blob/master/README.md) page.

### CODE REVIEW

File: Translations/ru/SCP079_HUD.txt [hunk 1]
```diff
@@ -71,4 +71,24 @@ SCP-079 не имеет возможности прямого боя или пе
...
+ВЫБРАННЫЕ ЗОНЫ: [cnt]
+ВЫБОР ЗОН ДЛЯ СКАНИРОВАНИЯ
+Результат сканирования: $Стационарные цели не обнаружены$.
+Повторная попытка через [time] секунд...
+Результат сканирования: $[role]$ обнаружен на ‘$[camera_name]$’ ([zone]).
+СЛЕДУЮЩЕЕ СКАНИРОВАНИЕ: [time] с.
+Сканирование...
+ПРОЦЕДУРА СКАНИРОВАНИЯ ПРИОСТАНОВЛЕНА
+ВЫБЕРИТЕ КОМАНДЫ ДЛЯ ОБНАРУЖЕНИЯ
+Сканер Нарушений обнаруживает цели, длительное время пребывающие в одной зоне.
```
Comment (DogooSlayer) [inline Translations/ru/SCP079_HUD.txt:93]: Лучше перевести так: "Сканер Нарушений обнаруживает цели, которые длительное время пребывают в одном и том же месте".

**Review by DogooSlayer — COMMENTED** [review]
Overall адекватный перевод. Не всматривался на 100%, но что увидел - то и прокомментировал. В основном проверял только новый текст. Возможно потребуется полная перепроверка всего текста, но это как-нибудь в другой раз.

File: Translations/ru/LoadingHints.txt [hunk 2]
```diff
@@ -29,35 +32,53 @@ Micro H.I.D. требуется около 5 секунд на зарядку п
 Micro H.I.D. очень громкий во время зарядки. Используйте его тогда, когда у ваших целей нет возможности убежать.
 Зарядка Micro H.I.D. сопровождается устрашающими звуками, которых достаточно, чтобы разрушить планы ваших врагов.
 Бутылка SCP-207 не только увеличивает вашу скорость, но и частично восстанавливает ваше здоровье.
-Можно выпить до четырёх бутылок SCP-207, что значительно увеличит вашу скорость.
+Можно употребить до четырёх бутылок SCP-207, что значительно увеличит вашу скорость.
```
[the remark this answers is not in the document]
Comment (bladuk) [reply Translations/ru/LoadingHints.txt:35]: В оригинальном переводе геймдизайнеры заменили слово "drunk" на "consumed".

**Review by DogooSlayer — COMMENTED** [review]
The texts looks better now. Though a full and deeper look into all translation texts would be great.

**Pull request merged**